### PR TITLE
finished homework

### PR DESCRIPTION
**Pull request by fahad5**
<html>

<head>
        <script type="text/javascript" src="http://code.jquery.com/jquery-3.3.1.min.js"></script>
        <link rel="stylesheet" type="text/css" href="weather.css">.

</head>
<body>
<div id="div1">
        

<input type="text" id="input"/>
<input type="submit" name="Get Data" id="getDataButton" value="Get weather">
</div>
<div id="div">

</div>
<span id="span">

</span>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="weather.js" type="text/javascript" charset="utf-8" async defer></script>
</body>
</html> 
   

body{

    margin: 0 ;

}

#div {
    margin: 0 auto ;
    align-content: center;
    /* background: red ; */
    width: 200px ;
    height: 200px;
}

#div1 {
    margin: 0 auto ;
    align-content: center;
    width: 200px ;
    height: 50px;
}

#span {

    color: black ;
} 

// var cityName = $("#input").val();
var city =document.getElementById("input")
var cityName = ""
// var cityName =  $("#input").attr('value')

// var weatherUrl = 'https://api.openweathermap.org/data/2.5/weather';
// var userinput = 'London';
// var appid = '7dccb07991a7f8cfca8e8e36aa8ebe25';
// var weatherURL = "http://api.openweathermap.org/data/2.5/weather?q=" + cityName + "&units=metric&appid=7dccb07991a7f8cfca8e8e36aa8ebe25";

//  Create instance of XMLHTTPRequest
$('#getDataButton').on('click',function(){
    console.log($(city).val())
    cityName = $(city).val()

    $.ajax({
        url: "http://api.openweathermap.org/data/2.5/weather?q=" + cityName + "&units=metric&appid=7dccb07991a7f8cfca8e8e36aa8ebe25",
        type: "GET",
        data: {
            // q: cityName,
            // appid: appid
        },
        success: function (response) {
            console.log(response);
            $('#span').html('<li>' + fahad +  '</li>');

        }

    });

    });

### CODE REVIEW

**Comment by morojAlh** [comment]
Good job 🌟
Just you need to append city name and some of the weather data to the DOM.

**Comment by fahad5** [comment]
Thank you so much moron for reviewing my code , i did change the code


On 14 Shaʻban 1440 AH at 5:15:47 PM, Moroj Alharbi (notifications@github.com)
wrote:

Good job 🌟
Just you need to append city name and some of the weather data to the DOM.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/misk-jsd2/08-ajax-and-apis/pull/20#issuecomment-484908755>,
or mute the thread

**Comment by fahad5** [comment]
Moroj*


On 14 Shaʻban 1440 AH at 7:15:47 PM, fahad alajmi (falajmi2012@gmail.com)
wrote:

Thank you so much moron for reviewing my code , i did change the code


On 14 Shaʻban 1440 AH at 5:15:47 PM, Moroj Alharbi (notifications@github.com)
wrote:

Good job 🌟
Just you need to append city name and some of the weather data to the DOM.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/misk-jsd2/08-ajax-and-apis/pull/20#issuecomment-484908755>,
or mute the thread
